### PR TITLE
Upgrade patch which solves issue #2582

### PR DESCRIPTION
**Pull request by JJJ4032002**
#### Because:
 
This Solves issue [#2582](https://github.com/TheOdinProject/theodinproject/issues/2582)

#### This commit

1.  Added styles to various tags.
2. Made it look as identical as the result picture given in the issue.

#### Checklist
 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/theodinproject/wiki/Contributing-Guide)?
 - [x] You have verified the tests and linters all pass against your changes?
 - [ ] You have included automated tests for your changes where 

 This is my first time contibuting here so i do not know where to add tests. If i have to include one please let me know

### CODE REVIEW

**Review by 01zulfi — CHANGES_REQUESTED** [review]
This is amazing :rocket: 

A couple of things.

Can you add a title to the webage by adding this line at the top of the file:

~~~ruby
<%= title('Installation Lessons Appendix') %>
~~~

File: app/views/lessons/installation_lessons/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+  <h1 class="page-heading-title text-center">Installation Lessons Appendix</h1>
 
   <ul>
+   
     <% @lessons.each do |lesson|%>
-     <li><%= link_to lesson.title, path_course_lesson_path(lesson.path, lesson.course, lesson) %></li>
+      
+      
+ 
+       <li> <p class = "text-1xl text-center m-3 text-gray-600"><%= link_to lesson.title, path_course_lesson_path(lesson.path, lesson.course, lesson), class: "underline hover:no-underline hover:text-gray-500 hover:border-gray-300" %> </p></li>
```
Comment (01zulfi) [inline app/views/lessons/installation_lessons/index.html.erb:10]: Another oversight from my part. I realized we don't need the `hover:border-gray-300` class here. Can you remove it?

Comment (JJJ4032002) [reply app/views/lessons/installation_lessons/index.html.erb:10]: Ok will do

File: app/views/lessons/installation_lessons/index.html.erb [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+   
 
   <ul>
    
     <% @lessons.each do |lesson|%>
       
       
  
-       <li> <p class = "text-1xl text-center m-3 text-gray-600"><%= link_to lesson.title, path_course_lesson_path(lesson.path, lesson.course, lesson), class: "underline hover:no-underline hover:text-gray-500 hover:border-gray-300" %> </p></li>
+       <li> <p class = "text-1xl text-center m-3 text-gray-600"><%= link_to lesson.title, path_course_lesson_path(lesson.path, lesson.course, lesson), class: "underline hover:no-underline hover:text-gray-500" %> </p></li>
```
Comment (JJJ4032002) [inline app/views/lessons/installation_lessons/index.html.erb:12]: @01zulfi Please do check the changes.

Comment (01zulfi) [reply app/views/lessons/installation_lessons/index.html.erb:12]: looks good to me 👍

**Comment by KevinMulhern** [comment]
Looks like the feedback was resolved, but just want to double-check if you're happy for this to be merged @01zulfi?

**Comment by 01zulfi** [comment]
@KevinMulhern Thanks for checking. Yes, I'm happy with it being merged

**Review by CouchofTomato — APPROVED** [review]

**Comment by CouchofTomato** [comment]
Thanks @JJJ4032002

**Pull request merged**